tests/test_onboarding.py

```diff
@@ -0,0 +1,47 @@
+from mollie.api.objects.onboarding import Onboarding
+from mollie.api.objects.organisation import Organisation
+
+ORGANISATION_ID = 'org_12345678'
+
+
+def test_get_onboarding(client, response):
+    """Retrieve onboarding."""
+    response.get('https://api.mollie.com/v2/onboarding/me', 'onboarding_me')
+
+    onboarding = client.onboarding.get('me')
+    assert isinstance(onboarding, Onboarding)
+
+    assert onboarding.name == 'Mollie B.V.'
+    assert onboarding.signed_up_at == '2018-12-20T10:49:08+00:00'
+    assert onboarding.status == 'completed'
+    assert onboarding.can_receive_payments is True
+    assert onboarding.can_receive_settlements is True
+
+    assert onboarding.is_needs_data() is False
+    assert onboarding.is_in_review() is False
+    assert onboarding.is_completed() is True
+
+
+def test_update_onboarding(client, response):
+    """Update onboarding.."""
+    response.patch('https://api.mollie.com/v2/onboarding/me', 'empty', 204)
+
+    data = {
+        'profile': {
+            'categoryCode': '6012'
+        }
+    }
+
+    onboarding = client.onboarding.update(resource_id='me', data=data)
```

Comment:
https://docs.mollie.com/reference/v2/onboarding-api/submit-onboarding-data Documents only POST, not PUT/PATCH.

Reply:
I changed this

Reply:
Could you also update the test name and the docstring? you are creating data, not updating it, I think.

Reply:
i updated this